examples/demos/client_merge_demo/main.go

```diff
@@ -47,7 +50,7 @@ func showClientMerge() error {
 	getResp, err := c.SyntheticsAdmin.ListAgents(context.Background(), &syntheticspb.ListAgentsRequest{})
```

Comment:
Indentation of the line above seems to be of.

Reply:
Good indentation is ensured by CI. The issue is that GH UI displays tabs inconsistently.